poll/poll.py

```diff
@@ -560,26 +562,27 @@ class SurveyBlock(PollBase):
     block_name = String(default='Poll')
```

Comment:
@asadiqbal08 "Poll" (and "Survey" in L559 above) should probably be marked for translation as well.